tests/e2e/tools/lib/editor.js

```diff
@@ -64,10 +66,36 @@ async function removeAllPostTypes({editor}) {
   }
 }
 
+/**
+ * Insert new block into page using the block inserter
+ *
+ * @param {Object} page             - The page object used to interact with the page.
+ * @param {Object} params           - Additional block's parameters.
+ * @param {Object} params.blockName - The name of the block.
+ * @param {Object} params.namespace - The namespace to search if it is needed.
+ * @return {Promise<void>}          - Playwright Locator
+ */
+const searchAndInsertBlock = async (page, {blockName, namespace = ''}) => {
```

Comment:
In our other functions, we set the first parameter as an object (so here it would be `{editor}` for example) but the other parameters not. Maybe for consistency we should do the same here?

Reply:
I was thinking `({page}, blockName, namespace = '')`, what do you think? Also the docs should be updated accordingly

Reply:
I'd prefer to use `{blockName, namespace = ''}` as `params` and be more consistent with the rest of functions.

Reply:
The rest of the functions don't use an object as second parameter 🤔

Reply:
Oh you're correct!